server/channels/app/export.go

```diff
@@ -491,6 +504,61 @@ func (a *App) exportAllUsers(ctx request.CTX, job *model.Job, writer io.Writer,
 	return profilePictures, nil
 }
 
+func (a *App) exportAllBots(ctx request.CTX, job *model.Job, writer io.Writer, includeProfilePictures bool) ([]string, *model.AppError) {
+	afterId := strings.Repeat("0", 26)
+	cnt := 0
+	profilePictures := []string{}
+
+	for {
+		bots, err := a.Srv().Store().Bot().GetAllAfter(1000, afterId)
+		if err != nil {
+			return profilePictures, model.NewAppError("exportAllBots", "app.user.get.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
+		}
+
+		if len(bots) == 0 {
+			break
+		}
```

Comment:
Isn't it more efficient to check for `len(bots) < 1000` at the end, and then break the loop? Not a big deal, it's just another SQL query. But seems a bit weird.